spec/statements_spec.rb

```diff
@@ -1,7 +1,7 @@
 module NoSE
   shared_examples 'a statement' do
     it 'tracks the range field' do
-      expect(statement.range_field).to eq tweet['Timestamp']
+      expect(statement.range_fields.first).to eq tweet['Timestamp']
     end
 
```

Comment:
I would probably change this to `expect(statement.range_fields).to eq [tweet['Timestamp']]` to be precise